test/correctness/simd_op_check_wasm.cpp

```diff
@@ -387,11 +387,11 @@ class SimdOpCheckWASM : public SimdOpCheckTest {
                 // check("v128.load32_zero", 2 * w, in_u32(0));
                 // check("v128.load64_zero", 2 * w, in_u64(0));
 
-                // Load vector with identical lanes
-                check("v128.load8_splat", 16 * w, in_u8(0));
-                check("v128.load16_splat", 8 * w, in_u16(0));
-                check("v128.load32_splat", 4 * w, in_u32(0));
-                check("v128.load64_splat", 2 * w, in_u64(0));
+                // Load vector with identical lanes generates *.splat.
+                check("i8x16.splat", 16 * w, in_u8(0));
```

Comment:
This will probably need to have a check for `internal::llvm_version()` so that tests running against LLVM15 (which we still test on main) will pass, there should be an example of that elsewhere in one of the simd_op_checks